src/marqo/tensor_search/validation.py

```diff
@@ -406,45 +415,47 @@ def validate_dict(field: str, field_content: Dict, is_non_tensor_field: bool, ma
                     f"The field {field} is a map field and only supported for indexes created with Marqo 2.9.0 or later. "
                     f"See {marqo_docs.map_fields()} and {marqo_docs.mappings()}."
                 )
-            
+
             field_content = validate_map_numeric_field(field_content)
 
     return field_content
 
+
 def validate_map_numeric_field(field_content):
-     """
-     Validates the field content if it is a map field (dict)
-     Args:
-         field_content: the field content
-     Returns:
-         field_content if the validation passes
-     """
-
-     # Validate that the field content is a dict
-     if not isinstance(field_content, dict):
-         raise InvalidArgError(
-             f"The field content `{field_content}` is of type `{type(field_content).__name__}`, which is not a valid type for a map field."
-             f"A map field must be a dictionary."
-         )
-
-     # Validate that the dict is only of one level
-     if any(isinstance(v, dict) for v in field_content.values()):
-         raise InvalidArgError(
-             "Nested dictionaries are not allowed in map fields. Each value must be a single int, float, or double."
-         )
-
-     # Validate that the values of the dict are int, float, or double
-     for key, value in field_content.items():
-         if not isinstance(key, str):
-             raise InvalidArgError(
-                 f"Key `{key}` in map field is not a string. All keys must be strings."
-             )
-         if not isinstance(value, (int, float)):
-             raise InvalidArgError(
-                 f"Value `{value}` for key `{key}` in map field is not of type int or float."
-             )
-
-     return field_content
+    """
```

Comment:
format change?

Reply:
yep, I believe this just changed spaces from 5 to 4.

Reply:
yep, I believe this just changed number of spaces from 5 to 4